tests/gnrc_sixlowpan_frag/main.c

```diff
@@ -32,7 +32,7 @@
 #define TEST_NETIF_IFACE        (9)
 #define TEST_TAG                (0x690e)
 #define TEST_PAGE               (0)
-#define TEST_RECEIVE_TIMEOUT    (100U)
+#define TEST_RECEIVE_TIMEOUT    (1000U)
```

Comment:
Huh?

Reply:
This timeout is defined in `us`, but the `gnrc_sixlowpan` now works with `ms`. This may jitter up to `1ms`. I've observed tests sometimes failing with `100us` timeout. Or doesn't that make sense to you?

Reply:
This disappeared by changing to ztimer_msec

Reply:
Then why not switch to `ZTIMER_MSEC` outright? I'm just confused: If you switch the value to be >=1ms, why not use the millisecond timer, especially if it stabilizes the test.

Reply:
Yes, I've already done this. I just didn't want to change to much at a time ... but I guess it's required to have a clean change ;-)